psscriptanalyzer/Backup-Database.psm1

```
@@ -0,0 +1,40 @@
+    $timestamp = Get-Date -Format "yyyyMMdd-HHmmss"
+    $backupFile = "$Database-$timestamp.bak"
+    $target = Join-Path $Global:BackupRoot (Join-Path $Database $backupFile)
+    $tempPath = Join-Path $env:TEMP "stage"
+    $command = "sqlcmd -S localhost -U $Username -P $Password -Q `"BACKUP DATABASE [$Database] TO DISK='$target'`""
+    Invoke-Expression $command
+    Write-Host "backup written to $target"
+}
+
+function Verify-Backups {
+  $stale = Get-Backups -Database "demoapp_prod"
+  if ($stale -eq $null) {
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)